modules/cli/src/test/java/org/apache/ignite/cli/commands/cliconfig/CliConfigCreateProfileCommandTest.java

```diff
@@ -0,0 +1,113 @@
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import jakarta.inject.Inject;
+import org.apache.ignite.cli.commands.CliCommandTestBase;
+import org.apache.ignite.cli.config.ProfileNotFoundException;
+import org.apache.ignite.cli.config.ini.IniConfigManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CliConfigCreateProfileCommandTest extends CliCommandTestBase {
+    @Inject
+    TestConfigManagerProvider configManagerProvider;
+
+    @Override
+    protected Class<?> getCommandClass() {
+        return CliConfigCreateProfileCommand.class;
+    }
+
+    @BeforeEach
+    void configManagerRefresh() {
+        configManagerProvider.configManager = new IniConfigManager(TestConfigManagerHelper.createSectionWithInternalPart());
+    }
+
+    @Test
+    public void testWithoutProfileName() {
+        execute();
+
+        assertAll(
+                this::assertOutputIsEmpty,
+                () -> assertErrOutputContains("Missing required option: '--name=<profileName>")
+        );
+    }
+
+    @Test
+    public void testProfileCreation() {
+        execute("--name profileName");
+
+        assertAll(
+                this::assertErrOutputIsEmpty,
+                () -> assertOutputContains("Profile profileName was created successfully"),
+                () -> assertThat(configManagerProvider.get().getCurrentConfig().getName()).isNotEqualTo("profileName")
+        );
+    }
+
+    @Test
+    public void testProfileCopyFrom() {
+        execute("--name profileName --copy-from database");
+
+        assertAll(
+                this::assertErrOutputIsEmpty,
+                () -> assertOutputContains("Profile profileName was created successfully"),
+                () -> assertThat(configManagerProvider.get().getConfig("profileName").getAll()).containsAllEntriesOf(
+                        configManagerProvider.get().getConfig("database").getAll())
+        );
+    }
+
+    @Test
+    public void testCopyFromNonExist() {
+        execute("--name profileName --copy-from notExist");
+
+        assertAll(
+                () -> assertExitCodeIs(1),
+                () -> assertErrOutputContains("Profile notExist not found"),
+                this::assertOutputIsEmpty,
+                () -> assertThatThrownBy(() -> configManagerProvider.get().getConfig("profileName"))
+                        .isInstanceOf(ProfileNotFoundException.class)
+        );
+    }
+
+    @Test
+    public void testProfileActivate() {
+        execute("--name profileName --activate");
+
+        assertAll(
+                () -> assertOutputContains("Profile profileName was created successfully"),
+                this::assertErrOutputIsEmpty,
+                () -> assertThat(configManagerProvider.get().getCurrentConfig().getName()).isEqualTo("profileName")
+        );
+    }
+
+    @Test
+    public void testCreateExistedProfile() {
+        execute("--name profileName");
+        execute("--name profileName");
```

Comment:
In order to test double execution, it is better to do it step by step: 
1) create a profile
2) check the profile is created
3) create the same profile again
4) check the error